GETTING_STARTED.md

```diff
@@ -1,6 +1,9 @@
 # Getting Started
 
-## Auth - Connect to GDN
+## 🆘 Macrometa Support
+If you have any trouble or need help while using SDK please contact [support@macrometa.com](mailto:support@macrometa.com).
```

Comment:
Should this be visible in PyPi Read me as well.. basically should we add it in README or gettingstarted?

Reply:
I agree! Will add 👍